internal/pkg/stream/ecs.go

```diff
@@ -123,6 +132,18 @@ func (s *ECSDeploymentStreamer) Fetch() (next time.Time, done bool, err error) {
 		return next, false, fmt.Errorf("fetch service description: %w", err)
 	}
 	s.retries = 0
+	var alarms []cloudwatch.AlarmStatus
+	if out.DeploymentConfiguration != nil && out.DeploymentConfiguration.Alarms != nil {
```

Comment:
Should we also be checking ` aws.BoolValue(out.DeploymentConfiguration.Alarms.Enable)`?